lib/crowbar/client/request/rest.rb

```diff
@@ -36,7 +36,7 @@ def initialize(options = {})
               url
             ].join(""),
             user: user,
```

Comment:
Don't we need to escape the user too?

Reply:
Do we support special characters in usernames? Reserved chars are: `;/?:@&=+$,[]`.

Reply:
I don't think we block them, so I guess we do support them there, yes.

Reply:
done.